src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.scss

```diff
@@ -25,6 +28,7 @@
   &__room-title {
     position: absolute;
     left: 50%;
+    bottom: -25px;
```

Comment:
Why the arbitrary number instead of something like 1em or 2em? There are two helper functions that I try to use when positioning or spacing text.

```scss
@function font-size--parent($scale: 1) {
  @return $scale * 1em;
}

@function font-size--root($scale: 1) {
  @return $scale * 1rem;
}
```
Any problem of adding `top:` instead of `bottom:`?

Reply:
fixed in d99a2c6e3c25c95b885194d2afba33e535beff2a